server/src/farcaster.ts

```diff
@@ -122,6 +128,14 @@ export const lookupFid = async (fid: number): Promise<FarcastUserInfo> => {
   return { fid, owner, username, timestamp: timestamp * 1000 }
 }
 
+export const mint = async (
+  id: number,
+  amount: number,
+  data: string
+): Promise<ContractTransaction> => {
+  return await dcReward.mint(config.mintAccount, id, amount, data)
```

Comment:
should mint chosen token id (preconfigured, corresponding to $COUNTRY, $MAP, $B) to user's account (owner of id)